x/auth/types/params.go

```diff
@@ -10,7 +10,7 @@ import (
 
 // Default parameter values
 const (
-	DefaultMaxMemoCharacters      uint64 = 256
+	DefaultMaxMemoCharacters      uint64 = 128
```

Comment:
I think we can explore how the memo is currently used. And limit the number to the minimum necessary. also, In celestia people can exchange messages directly in the blobs instead of the memo field.